platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/SimpleMapActivity.java

```diff
@@ -20,6 +22,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mapView = (MapView) findViewById(R.id.mapView);
     mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(new OnMapReadyCallback() {
+      @Override
+      public void onMapReady(MapboxMap mapboxMap) {
+        mapboxMap.getUiSettings().setCompassFadeFacingNorth(false);
```

Comment:
Changes in `SimpleMapActivity.java` seem unnecessary.

Reply:
should be removed, that was a test for a different issue